### PR TITLE
Tasks component

### PR DESCRIPTION
**Pull request by Fizzadar**
Currently adds a basic tasks dashboard:

Screenshot 2019-11-12 at 11 31 36

TODO:

+ [x] show/view sub-tasks
+ [x] pretty format running time without breaking sorting
+ [x] periodic updating / manual update button
+ [x] controls to show/hide child tasks

Closes: #15 
Closes: #16 
Closes: #19

### CODE REVIEW

File: frontend/src/TasksDashboard.jsx [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+    loadNodes,
+}
+
+class TasksDashboard extends React.Component {
+    static propTypes = {
+        tasks: PropTypes.shape({
+            loadingState: PropTypes.oneOf(['NOT_LOADED', 'LOADING', 'LOADED', 'ERROR']).isRequired,
+            data: PropTypes.object,
+            error: PropTypes.object,
+        }).isRequired,
```
Comment (gchazot) [inline frontend/src/TasksDashboard.jsx]: Feels like this shape could be refactored

File: README.md [hunk 2]
```diff
@@ -19,3 +19,13 @@ And then to start the webpack server and follow the server logs:
...
 ```
+
+### Pointing at another cluster
+
+To look at another cluster in development, first make a copy of `fixtures/dev_cluster_config.json` and add it to the development Elasticsearch instance like so:
+
+```sh
+curl -X PUT http://dev.edtd.net:26950/.elasticsurgery-clusters/_doc/<SLUG> \
+    -H 'Content-Type: application/json' \
+    -d@my_json_config.json
```
Comment (gchazot) [inline README.md]: We should add a script to simplify this, and also use it in the `migrations` section of `kubetools.yaml`

File: frontend/src/TasksDashboard.jsx (hunk 1 above)
Comment (Fizzadar) [reply frontend/src/TasksDashboard.jsx]: Agreed! https://github.com/EDITD/ElasticSurgery/pull/18/commits/13d9e67cccf837531db619baa90cc71b1c79e110

File: README.md (hunk 2 above)
Comment (Fizzadar) [reply README.md]: This already happens during ktd up for the dev cluster - the clusters UI should remove the need for this entirely once implemented, so a temporary readme addition!

**Comment by Fizzadar** [comment]
Thanks for the review @gchazot!

**Pull request merged**